.github/workflows/ci.yml

```diff
@@ -251,11 +251,10 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Install mesh-cli
         run: curl -sSfL https://raw.githubusercontent.com/coinbase/mesh-cli/master/scripts/install.sh | sh -s
-      - name: Generate keygen
-        run: ./gradlew keygen:run --args='-k ../core/node.ks -p ""'
-      - name: Build Node
-        run: ./gradlew build
       - name: Run Node in the background
+        env:
```

Comment:
This also fixes the issue observed earlier
```
Command Failed: unable to initialize asserter for fetcher: /network/status not attempting retry: /network/status response is invalid: current block timestamp 0 is invalid: timestamp is before 01/01/2000
```
It seems like a node was not processing any transactions.